packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -271,15 +264,15 @@ export class VaTextInput {
     const ariaDescribedbyIds =
       `${messageAriaDescribedby ? 'input-message' : ''} ${
         error ? 'input-error-message' : ''} ${
-        charcount && maxlength ? 'charcount-message' : ''}`.trim() || null; // Null so we don't add the attribute if we have an empty string
```

Comment:
This is a bit confusing. The PR title says deprecating max length but `maxlength` still exists (by naming convention at least). Can you add some screenshots and details to the PR description so it's clearer exactly what is being removed and is remaining?

Reply:
On the v1 Storybook page, I still see an example for Max Length too:

![Screenshot 2024-01-17 at 1 26 23 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/9472b807-4e1a-46a2-9001-5d4262f50431)

Reply:
The Max Length variation for V3 is the only thing being deprecated, it's being replaced by the Character Count version. The maxlength prop is still needed, but the charcount prop is not, as it will default to showing the character count if a max length is specified.

V1 isn't being changed at this time. I agree, it's a little confusing.